prac_06/guitar.py

```diff
@@ -0,0 +1,35 @@
+"""
+CP1404/CP5632 Practical - Suggested Solution
+Guitar class
+"""
+CURRENT_YEAR = 2022
+VINTAGE_NUMBER = 50
+
+
+class Guitar:
+    """Guitar class for storing details about a Guitar."""
+
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        """Return a string representation of a Guitar."""
+        return f"{self.name} ({self.year}) : ${self.cost}"
+
+    def get_age(self):
+        """Get the age of a guitar based on CURRENT_YEAR."""
+        age = CURRENT_YEAR - self.year
+        return age
```

Comment:
I like how this reads well with returning age, could you have done this in a single line instead?